closed/make/modules/openjceplus/Lib.gmk

```diff
@@ -60,13 +62,18 @@ endif # OPENJCEPLUS_JGSKIT_PLATFORM
 
 compile-libs :
 	@$(ECHO) Compiling OpenJCEPlus native code
-	export \
-			$(OPENJCEPLUS_VS_LIB) \
-			GSKIT_HOME=$(OPENJCEPLUS_GSKIT_HOME) \
-			JAVA_HOME=$(OPENJCEPLUS_BOOT_JDK) \
-			JCE_CLASSPATH=$(OPENJCEPLUS_JCE_CLASSPATH) \
-			PLATFORM=$(OPENJCEPLUS_JGSKIT_PLATFORM) \
-		&& $(MAKE) -j1 -C $(OPENJCEPLUS_JGSKIT_MAKE_PATH) -f $(OPENJCEPLUS_JGSKIT_MAKE) all
+  $(EXPORT_COMPILER_ENV_VARS) \
```

Comment:
This line must begin with a tab (else it isn't part of the script for `compile-libs`).

Reply:
Sorry. The editor added spaces instead of tab and I missed it. Fixed now.